index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Main Css file -->
+    <link rel="stylesheet" href="css/master.css" />
+    <!-- Render all elements normally -->
+    <link rel="stylesheet" href="css/Normalize.css" />
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio Website: Mohammed Raad</title>
+  </head>
+  <body>
+    <!-- Toolbar start -->
+    <nav id="nav">
+      <div class="nav-container container">
+        <div class="name">Mohammed Raad</div>
+        <div class="links">
+          <span class="icon">
+            <span></span>
+            <span></span>
+            <span></span>
+          </span>
+        </div>
+      </div>
+    </nav>
+    <!-- Toolbar end -->
+    <!--  headline section start -->
+    <section id="headline">
+      <div class="headline-container container">
```

Comment:
- [ ] Kindly reduce the space between the toolbar and the title so it matches the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)